fair-api.yaml

```diff
@@ -40,6 +40,43 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /endpoints:
+    x-indicator: False
+    get:
+      summary: Return all the possible endpoints in a JSON format
+      tags:
+        - RDA indicators
+      operationId: api.rda.endpoints
+      parameters:
+      - in: query
+        name: plugin
+        schema:
+          type: string
+        required: False
+        description: The plugin you want the enpoint for
+      responses:
+        '200':
+          description: Expected response to a valid request
+          content:
+            application/json:
+             schema:
+               type: string
+        '204':
```

Comment:
204 refers to "no content" and cannot handle errors. The response code to a wrong plugin request should be 404